server/sonar-process/src/main/java/org/sonar/process/cluster/ClusterObjectKeys.java

```diff
@@ -21,32 +21,35 @@
 package org.sonar.process.cluster;
 
 /**
- * This interface holds all object keys accessible via Hazelcast
+ * This class holds all object keys accessible via Hazelcast
  */
-public interface ClusterObjectKeys {
+public class ClusterObjectKeys {
```

Comment:
if `ClusterObjectKeys` is never instantiated, because it's only a holder for constants, then the class should be final and it's default constructor private (don't forget to add a comment in the constructor's body, otherwise SQ will raise an issue)